Object oriented system lab 1

solovdaria opened the pull request.

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

solovdaria committed:
Delete Object oriented system.sln

solovdaria committed:
Delete Object oriented system.exe

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

solovdaria committed:
Delete Object oriented system.sln

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

Документація: 
file:///C:/Users/user/source/repos/Object%20oriented%20system/Object%20oriented%20system/documentation/html/index.html

solovdaria committed:
Update Object oriented system.cpp

dt-iss reviewed (commented):
The project structure is ugly. It should be revised and restructured.
Note that, according to the task, different kinds of trees should be implemented.

solovdaria committed:
Add files via upload

file:///C:/Users/user/source/repos/Object%20oriented%20system/Object%20oriented%20system/documentation/html/files.html Документація

solovdaria committed:
Add files via upload

file:///C:/Users/user/source/repos/Object%20oriented%20system/Object%20oriented%20system/documentation/html/files.html Документація

solovdaria committed:
Add files via upload

solovdaria committed:
Delete MyTree.h

solovdaria committed:
Delete Object oriented system.cpp

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

solovdaria committed:
Delete BinarySearchTree.h

solovdaria committed:
Delete BinaryTree.h

solovdaria committed:
Delete Dice.cpp

solovdaria committed:
Delete Dice.h

solovdaria committed:
Delete N_aryTree.h

solovdaria committed:
Delete NodeList.h

solovdaria committed:
Delete Object_or_sys2.cpp

solovdaria committed:
Delete node.h

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

solovdaria committed:
Delete pch.cpp

solovdaria committed:
Delete pch.h

solovdaria committed:
Delete BinarySearchTree.h

solovdaria committed:
Delete BinaryTree.h

solovdaria committed:
Delete Dice.cpp

solovdaria committed:
Delete Dice.h

solovdaria committed:
Delete N_aryTree.h

solovdaria committed:
Delete NodeList.h

solovdaria committed:
Delete SetOfDice.cpp

solovdaria committed:
Delete SetOfDice.h

solovdaria committed:
Delete node.h

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

solovdaria committed:
Add files via upload

UML added
